internal/webserver/server.go

```diff
@@ -239,3 +239,16 @@ func (webserver *WebServer) StartHTTPServer(errChannel chan error) {
 		}
 	}()
 }
+
+// StartHTTPSServer starts the https server
+func (webserver *WebServer) StartHTTPSServer(errChannel chan error) {
```

Comment:
Lots of repeated code doing it this way. Better to pass the protocol as a new parameter and use it in a conditional for just the call to the appropriate ListenAndServe

Reply:
hi Lenny, you're right. Tried to modify the code with minimal impact on existing code but was not a good idea. Can you check it now?

Reply:
That is much better. Thx